TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift

```diff
@@ -219,4 +222,18 @@ class TradeItSelectBrokerViewController: CloseableViewController, UITableViewDel
 
         return cell
     }
+
+    private func getRemoteLogo(forBroker broker: TradeItBroker, imageView: UIImageView) -> Bool {
```

Comment:
Can we do some refactoring and put this code inside the `TradeItSDK.brokerLogoService.getLogo` method instead ?